packages/components/src/components/input/input.stories.ts

```diff
@@ -69,731 +35,219 @@ export const Default = {
 };
 
 /**
- * Use the `label` attribute to give the input an accessible label. For labels that contain HTML, use the `label` slot instead.
+ * Use the `size` attribute to change the size.
  */
 
-export const Labels = {
-  args: overrideArgs([{ type: 'attribute', name: 'label', value: 'Label' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Size = {
+  render: () => html`
+    <div class="grid grid-cols-2 gap-12 content-end">
+      <sd-input size="lg" label="Label"> </sd-input>
+      <sd-input size="md" label="Label"> </sd-input>
+      <sd-input size="sm" label="Label"> </sd-input>
+    </div>
+  `
 };
 
 /**
- * Add descriptive help text to an input with the `help-text` attribute. For help texts that contain HTML, use the `help-text` slot instead.
+ * Use the `label` attribute to give the input an accessible label.
+ *
+ * For labels that contain HTML, use the `label` slot instead.
  */
 
-export const HelpText = {
-  name: 'Help Text',
-  args: overrideArgs([
-    { type: 'attribute', name: 'label', value: 'Nickname' },
-    { type: 'attribute', name: 'help-text', value: 'How would you like to be called?' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Label = {
+  render: () => html`
+    <div class="flex flex-row gap-12 items-baseline">
+      <sd-input label="Label" spellcheck></sd-input>
+      <sd-input spellcheck>
+        <div slot="label" class="text-lg">Label</div>
+      </sd-input>
+    </div>
```

Comment:
Please only add input size lg (default) here

![image](https://github.com/user-attachments/assets/23c04843-9116-460c-8e9d-bac1f9e73d41)

Reply:
Hey @yoezlem in this case is not a matter of sizes but there are two different examples because the user have two options:

- Using the attibute 
- Using the slot 

I though it would be usefull to have both examples for the code. Do you agree?